packages/rest/README.md

```diff
@@ -12,84 +12,161 @@ npm install @cadl-lang/rest
 
 ## Usage
 
-See [Rest section in the tutorial](../../docs/tutorial.md#rest-apis)
+```Cadl
+import "@cadl-lang/rest";
 
-## Decorators
-
-The `@cadl-lang/rest` library defines the following decorators:
-
-### `@parentResource`
-
-Syntax:
-
-```
-@parentResource(parentModelTypeReference)
+using Cadl.Http;
+using Cadl.Rest;
 ```
 
-`@parentResource` marks a model property with a reference to its parent resource type
-
-The first argument should be a reference to a model type which will be treated as the parent
-type of the target model type. This will cause the `@key` properties of all parent types of
-the target type to show up in operations of the `Resource*<T>` interfaces defined in this library.
+See [Rest section in the tutorial](https://github.com/microsoft/cadl/blob/main/docs/tutorial.md#rest-apis)
+
+## Library Tour
+
+`@cadl-lang/rest` library defines of the following artifacts:
+
+- [Models](#models)
+- [Decorators](#decorators)
+- [Interfaces](#interfaces)
+
+## Models
+
+- ### HTTP namespace
+  | Model                | Notes                                                                                                                                  |
+  | -------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+  | LocationHeader       | Location header                                                                                                                        |
+  | Response&lt;Status>  | &lt;Status> is numerical status code.                                                                                                  |
```

Comment:
are those `&lt;` on purpose? does `<` not work? make it harder to read the markdown

Reply:
No, < doesn't quite work in the table.